tools/test-setup.sh

```diff
@@ -85,7 +85,7 @@ if [ -f /etc/os-release ]; then
             # https://github.com/hashicorp/vagrant/issues/11020
             if grep -qi '^CentOS Linux release 8.2.*' /etc/centos-release ; then
                 # https://bugs.centos.org/view.php?id=17120
-                relver="$(cat /etc/centos-release | awk '{print $4}')"
+                relver="$(grep -v '^#' /etc/centos-release | awk '{print $4}')"
```

Comment:
Is this a change from shellcheck too ?

Reply:
yeah, https://www.shellcheck.net/wiki/SC2002 "Useless cat", but it's really nitpicking in my opinion.